ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/result-form-selector/dropdown.result-form-selector.view.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+define([
+    'marionette',
+    'underscore',
+    'jquery',
+    '../dropdown.view',
+    './dropdown.result-form-selector.hbs',
+    'component/result-form-selector/result-form-selector.view',
+    'js/store'
+], function (Marionette, _, $, DropdownView, template, ResultForms, store) {
+
+    return DropdownView.extend({
```

Comment:
✏️ So I messed around with this a bit. Didn't get it to work with a `createSimpleDropdown` either, but I did manage to get it to work with just
```
define([
    '../dropdown.view',
    './dropdown.result-form-selector.hbs',
    'component/result-form-selector/result-form-selector.view'
], function (DropdownView, template, ResultForms) {

    return DropdownView.extend({
        template: template,
        className: 'is-search-form-selector',
        componentToShow: ResultForms
    });
});
```

Also, we may want to consider making a new `.less` file for this component instead of piggybacking off the styles from `dropdown.search-form-selector.less` by using the same `is-search-form-selector` class on this component as opposed to a `is-result-form-selector` class

Reply:
This modification messed up the refresh after creation.  Going to leave it for now while I investigate more on why.